src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -109,7 +115,35 @@ public static class PrereleaseCompatibilityTransformer
             return transformedSarifLog;
         }
 
-        private static bool ApplyChangesFromTC25ThroughTC28(
+        private static bool ApplyChangesFromTC31(JObject sarifLog)
+        {
+            bool modifiedLog = UpdateSarifLogVersion(sarifLog);
+
+            var runs = (JArray)sarifLog["runs"];
+
+            if (runs != null)
+            {
+                foreach (JObject run in runs)
+                {
+                    if (run["results"] is JArray results)
+                    {
+                        foreach (JObject result in results)
+                        {
+                            string baselineState = (string)result["baselineState"];
+
+                            if ("existing".Equals(baselineState))
```

Comment:
I suggest `StringComparison.Ordinal`

Reply:
Ordinal is what's used for String.Equals. This is an important fact about .NET which is my argument for preferring not to make it explicit.

---
In reply to: [252001522](https://github.com/Microsoft/sarif-sdk/pull/1241#discussion_r252001522) [](ancestors = 252001522)